Task2/aga/TicTacToe/TicTacToe/TheGame.xaml.cs

```diff
@@ -0,0 +1,242 @@
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace TicTacToe
+{
+    /// <summary>
+    /// Interaction logic for TheGame.xaml
+    /// </summary>
+    public partial class TheGame : Window
+    {
+        public TheGame()
+        {
+            InitializeComponent();
+
+        }
+        Click click = new Click();
+        private bool[] clickedButtons = new bool[9];
+        private object winner = "";
+
+        private void A1_Click(object sender, RoutedEventArgs e)
+        {
+            Button b = (Button)sender;            
+            int turn = click.GetTurn();
+            int param = 0;
+
+            displayButton(turn, b, param);
+
+        }
+
+        private void A2_Click(object sender, RoutedEventArgs e)
+        {
+            Button b = (Button)sender;
+            int turn = click.GetTurn();
+            int param = 1;
+
+            displayButton(turn, b, param);
+        }
+
+        private void A3_Click(object sender, RoutedEventArgs e)
+        {
+            Button b = (Button)sender;
+            int turn = click.GetTurn();
+            int param = 2;
+
+            displayButton(turn, b, param);
+        }
+
+        private void B1_Click(object sender, RoutedEventArgs e)
+        {
+            Button b = (Button)sender;
+            int turn = click.GetTurn();
+            int param = 3;
+
+            displayButton(turn, b, param);
+        }
+
+        private void B2_Click(object sender, RoutedEventArgs e)
+        {
+            Button b = (Button)sender;
+            int turn = click.GetTurn();
+            int param = 4;
+
+            displayButton(turn, b, param);
+        }
+
+        private void B3_Click(object sender, RoutedEventArgs e)
+        {
+            Button b = (Button)sender;
+            int turn = click.GetTurn();
+            int param = 5;
+
+            displayButton(turn, b, param);
+        }
+
+        private void C1_Click(object sender, RoutedEventArgs e)
+        {
+            Button b = (Button)sender;
+            int turn = click.GetTurn();
+            int param = 6;
+
+            displayButton(turn, b, param);
+        }
+
+        private void C2_Click(object sender, RoutedEventArgs e)
+        {
+            Button b = (Button)sender;
+            int turn = click.GetTurn();
+            int param = 7;
+
+            displayButton(turn, b, param);
+        }
+        private void C3_Click(object sender, RoutedEventArgs e)
+        {
+            Button b = (Button)sender;
+            int turn = click.GetTurn();
+            int param = 8;
+
+            displayButton(turn, b, param);
+        }
+
+        private void displayTurn(int t)
+        {
+            Label l = diplayTurn;
```

Comment:
displayTurn is a name that you gave to a Label control in XAML file and you're using it in code behind. That's 2 sins in 1 line :-1: